lib/Elastica/Cluster.php

```diff
@@ -73,14 +73,7 @@ public function getResponse()
      */
     public function getIndexNames()
     {
-        $metaData = $this->_data['metadata']['indices'];
-
-        $indices = array();
-        foreach ($metaData as $key => $value) {
-            $indices[] = $key;
-        }
-
-        return $indices;
+        return array_keys($this->_data['metadata']['indices']);
```

Comment:
@ruflin, Yes exists test in \Elastica\Test\StatusTest::testGetIndexNames https://github.com/ruflin/Elastica/blob/master/test/lib/Elastica/Test/StatusTest.php#L37